Anomaly/Patches/AbilityDefs/Abilities_Anomaly.xml

```diff
@@ -39,8 +39,8 @@
 		<value>
 			<projectile Class="CombatExtended.ProjectilePropertiesCE">
 				<damageAmountBase>11</damageAmountBase>
```

Comment:
I don't understand why you've changed around the spike projectile AP values like this. Fingerspike does more AP than the Gorehulk, and Toughspike does more than both?

Reply:
It's not 20% less, it's 100% vs 15% armor penetration... the base spike projectile is was dictates the tough/trispike and fingerspike projectiles which same as the sanghuophage projectiles, have 100% AP in vanilla.